includes/class-wc-stripe-helper.php

```
@@ -2203,4 +2203,27 @@ public static function get_exit_survey_params( WC_Stripe_Account $account ): arr
+
+		// Stripe checkout sessions feature can only be available if:
+		// - PMC is enabled
+		// - OC Suite is enabled
+		// - Automatic capture is enabled (i.e. manual capture or later capture is disabled)
+		// If any of the above conditions are not met, the feature is not available.
+		if ( 'yes' !== $is_pmc_enabled || 'yes' !== $is_oc_enabled || 'yes' !== $is_automatic_capture_enabled ) {
+			return false;
+		}
+
+		return true;
+	}
```

PHPMD (2.15.0)

[error] The method is_checkout_sessions_available is not named in camelCase. (undefined)

(CamelCaseMethodName)